### DISCUSSION TITLE
Is there any means of changing GB to MB in preferences on MacOS?

### DISCUSSION THREAD

**Discussion by GaryElshaw** [transmission/transmission#2476] [Q&A]
I was wondering if there is anything in any of the Transmission config files where i can change GB's to MB's? Depending on a number of variables, i'd like to be able to see everything in MB's in the main window. IE, instead of:

1.61 GB, uploaded 1.49 GB...... - is it possible to have 1,61x MB, uploaded 1,49x MB?

**Comment by ckerr** [comment]
I don't think there's a preference setting for that; it would need a patch

**Reply by GaryElshaw** [reply]
> it would need a patch

Thanks Charles. I'm sure this has come up before, but what about the granularity of the percentage seen during download being available in upload? Essentially i'm trying to bridge the gulf that appears when trying to work out time. I spend a great deal more time seeding than downloading and the time estimates for seeding are just silly. When i learn to code, at the start of the seeding process, i'm going to replace 'remaining time....' with ¯\_(ツ)_/¯ until it's about 10 minutes from completion.
